spec/middleware/catch_json_parse_errors_spec.rb

```diff
@@ -10,7 +10,7 @@
     expect(curb.response_code).to eq 400
     expect(curb.content_type).to match(/application\/json/)
     expect(curb.body_str).to match(
-      'There was a problem in the JSON you submitted:'
+        { "status" => 400, "error" => "Invalid JSON"}.to_json
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.